scylla-cql/src/types/serialize/row.rs

```diff
@@ -125,13 +131,150 @@ pub enum ValueListToSerializeRowAdapterError {
     NoBindMarkerWithName { name: String },
 }
 
+/// A container for already serialized values. It is not aware of the types of contained values,
+/// it is basically a byte buffer in the format expected by the CQL protocol.
+/// Usually there is no need for a user of a driver to use this struct, it is mostly internal.
```

Comment:
Nit: conventionally, a rust docstring starts with a single summary sentence, followed by a blank line and a more detailed description.